confluent_kafka/avro/__init__.py

```diff
@@ -1,3 +1,20 @@
+#!/usr/bin/env python
```

Comment:
again, don't understand why this is executable

Reply:
afaict no one else important does this, and it's bordering on pointless. it didn't help me in any way having this here.